docs/ism/policies.md

```diff
@@ -516,6 +516,75 @@ The destination system **must** return a response otherwise the `error_notificat
 
 You can use the same options for `ctx` variables as the [notification](#notification) operation.
 
+## Sample template for rollover
```

Comment:
Should be like "Sample policy with ism_template"

This is still a policy, template is a field for it, and since ES also has template function, we should say this is ISM template to distinguish